eventrouter.go

```diff
@@ -154,6 +154,10 @@ func (er *EventRouter) updateEvent(objOld interface{}, objNew interface{}) {
 		glog.Error("Given object '%v' not v1.Event", objNew)
 		return
 	}
+	if eOld.ResourceVersion == eNew.ResourceVersion {
```

Comment:
This seems like a reasonable change, but does it actually solve the reported bug? Do we have reason to think the resource versions were the same?

Reply:
my recollection is "resourceversion" is global to all objects and ever increasing but I don't know we have evidence this solves the issue

Reply:
@jcantrill @alanconway 
https://github.com/kubernetes/community/blob/8cafef897a22026d42f5e5bb3f104febe7e29830/contributors/devel/controllers.md take a look at post # 9